include/lsst/afw/cameraGeom/Detector.h

```diff
@@ -210,17 +210,17 @@ class Detector {
      *
      * @note the CameraSysPrefix version needs the detector name, which is why this is not static.
      */
-    CameraPoint makeCameraPoint(geom::Point2D point,  ///< 2-d point
-                                CameraSys cameraSys   ///< coordinate system
+    CameraPoint makeCameraPoint(geom::Point2D const &point,  ///< 2-d point
+                                CameraSys const &cameraSys   ///< coordinate system
                                 ) const {
         return CameraPoint(point, cameraSys);
     }
 
     /**
      * Make a CameraPoint from a point and a camera system prefix
      */
-    CameraPoint makeCameraPoint(geom::Point2D point,             ///< 2-d point
-                                CameraSysPrefix cameraSysPrefix  ///< coordinate system prefix
+    CameraPoint makeCameraPoint(geom::Point2D const &point,             ///< 2-d point
+                                CameraSysPrefix const &cameraSysPrefix  ///< coordinate system prefix
```

Comment:
It's the result of running clang-format, which I prefer to hand-formatting. I don't like all its choices, but feel they're usually good enough, and our standards accept it.